mmaction/models/registry.py

```diff
@@ -6,3 +6,4 @@
 RECOGNIZERS = Registry('recognizer')
 LOSSES = Registry('loss')
 LOCALIZERS = Registry('localizer')
+DETECTORS = Registry('detector')
```

Comment:
What's the relation of this detector to mmdet's?

Reply:
This line is unused and will be removed.